migrations/V181__Add_MTO_Solution.sql

```diff
@@ -22,7 +22,7 @@ CREATE TABLE mto_solution(
     type MTO_SOLUTION_TYPE,
     facilitated_by MTO_FACILITATOR,
     status MTO_SOLUTION_STATUS NOT NULL,
-    risk_indicator MTO_RISK_INDICATOR,
+    risk_indicator MTO_RISK_INDICATOR NOT NULL DEFAULT 'ON_TRACK',
     is_draft BOOLEAN NOT NULL DEFAULT TRUE,  --TODO (mto) is this field needed?

```

Comment:
Can probably just nix this field and the associated `TODO`

Looking through the Figma mockups, there's a pretty clear indicator that milestones can be marked as Draft ([src](https://www.figma.com/design/BcI7pNwfkmwaDD2EGGTsYC/IT-Lead?node-id=3619-31283&node-type=frame&t=3WFlUET2ohiwUTT8-0)), but there's not a field for editing a solution ([src](https://www.figma.com/design/BcI7pNwfkmwaDD2EGGTsYC/IT-Lead?node-id=3267-98504&node-type=instance&t=3WFlUET2ohiwUTT8-0))

Reply:
Or, honestly, feel free to leave it if you don't want to touch this a bunch here, since I know we opted to push `solution` work out and focus this PR on `milestones`!

I'll leave the call to you, but I think we can pretty safely/confidently remove this field

Reply:
Thanks Clay! I'll just remove it here since we have all the context right now :D Thanks!

Reply:
Resolved in [781550b](https://github.com/CMS-Enterprise/mint-app/pull/1454/commits/781550b7c44442a244b34acb43d26c3f9a6d31d8)